src/xpcc/architecture/platform/board/stm32f103c8t6_msdb/stm32f103c8t6_msdb.hpp

```diff
@@ -0,0 +1,45 @@
+// coding: utf-8
+/* Copyright (c) 2016, Roboterclub Aachen e. V.
+* All Rights Reserved.
+*
+* The file is part of the xpcc library and is released under the 3-clause BSD
+* license. See the file `LICENSE` for the full license governing this code.
+*/
+// ----------------------------------------------------------------------------
+
+//
+// STM32F103C8T6 Minimum System Development Board (MSDB)
+//
+// Cheap and bread-board-friendly board for STM32 F1 series.
+// Sold for less than 2 USD on well known Internet shops from China.
+//
+
+#ifndef XPCC_STM32_F103C8T6_MSDB_HPP
+#define XPCC_STM32_F103C8T6_MSDB_HPP
+
+#include <xpcc/architecture/platform.hpp>
+
+using namespace xpcc::stm32;
+
+
+namespace Board
+{
+
+/// STM32F103 running at 72MHz generated from the external 8MHz crystal
+using systemClock = SystemClock<Pll<ExternalCrystal<MHz8>, MHz72> >;
```

Comment:
You may be able to [use the F103RB clock tree from the Nucleo board](https://github.com/roboterclubaachen/xpcc/blob/develop/src/xpcc/architecture/platform/board/nucleo_f103rb/nucleo_f103rb.hpp#L27-L87) and modify it to the C8 clock tree.